ui/components/multichain/pages/send/send.js

```diff
@@ -405,14 +406,16 @@ export const SendPage = () => {
             onClick={onCancel}
           />
         }
+        endAccessory={
+          remoteModeSupported && (
+            <SendAllowanceTooltip allowance={remoteSendAllowance} />
+          )
+        }
       >
         {t('send')}
       </Header>
       <Content>
-        {showRemoteSendBanner && (
-          <SendAllowanceBanner allowance={remoteSendAllowance} />
-        )}
-        <SendPageAccountPicker isRemoteModeEnabled={isRemoteSendPossible} />
```

Comment:
Can we also remove the `isRemoteModeEnabled` property from `SendPageAccountPicker`?